pkg/kubeapiserver/options/storage_versions_test.go

```diff
@@ -76,3 +71,36 @@ func TestGenerateStorageVersionMap(t *testing.T) {
 		}
 	}
 }
+
+func TestToPreferredVersionString(t *testing.T) {
```

Comment:
The function is very simple. A unit test doesn't seem necessary. WDYT?

Reply:
MapToString seems test well in others place, remove seems OK